### ISSUE TITLE
Update to 1.20.1 please

### ISSUE THREAD

**Issue by jghf1312** [ChiselsAndBits/FlatColoredBlocks#85]
i have 1.20.1 but, this mod only has for 1.19.2 and i want to update my mods so please make a 1.20.1 version
thank you
good mod

**Comment by MaryaneVih** [comment]
****

**Comment by MaryaneVih** [comment]
Legal 😎

**Comment by TheSatanicFlame** [comment]
Bump, We have a 1.20.4 version but 1.20.1 is the modded version of this update

**Comment by JurieJVN** [comment]
Agreed - Please add 1.20.1 for forge ♥

**Comment by DAVIDJONES6969** [comment]
I agree, I would really appriciate, if the flat coloured blocks mod was updated to 1.12.1, please add.
